docs/overview.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Overview"
+description: "OpenPipe is a streamlined platform designed to help product-focused teams train specialized LLM models as replacements for slow and expensive prompts."
+---
+
+## Who We Are
+
+We're a team of full-stack engineers and machine learning researchers who came together to streamline the process of integrating fine-tuned models into any application. We believe that specialized models will continue to outperform enormous multipurpose LLMs for decades to come, and our goal is to make the fine-tuning process accessible to everyone.
```

Comment:
Don't think docs are the right place to put this kind of stuff. Devs are turned off by stuff that sounds like marketing. This should focus more on what the experience of using openpipe is like from a dev's point of view, not our Big Vision